### ISSUE TITLE
Improve misplaced \noalign error description

### ISSUE THREAD

**Issue by besendorf** [jgm/pandoc#8650]
Sometimes when converting .md to pdf a `misplaced \noalign` LaTeX error occurs. See #859.
This error however is very unclear to a user because the line number it shows is a LaTeX line number and thus the user doesn't know where the error in his markdown file ist. Here another example:

```
Error producing PDF.
! Misplaced \noalign.
<recently read> \noalign 
                         
l.951 \toprule\noalign
```
My .md file has only ~500 lines

**Comment by jgm** [comment]
Correct. When you're using pandoc to produce a PDF, it is creating LaTeX and then compiling this with pdflatex.  If pdflatex raises an error, we report it, but of course the line numbers refer to the LaTeX intermediary file.  I don't think there's any good way to trace this back to a line number in the markdown file. However, you can use `--verbose` and it will print the entire LaTeX intermediary file to stderr; this will allow you to figure out where the problem lies.
